### PR TITLE
nvidia: Fix module compilation on 6.12 kernel

### PR DESCRIPTION
**Pull request by 1Naim**

### CODE REVIEW

**Comment by 1Naim** [comment]
NVIDIA users i choose you! (please test thx)

File: nvidia/nvidia-open-dkms/0007-Replace-PageSwapCache-for-6.12-kernel.patch [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
++#endif
+ 
+             // If the page is already allocated, it is most likely a mirrored
+             // page. Check to be sure it matches what we have recorded. The
+@@ -2748,7 +2757,7 @@ static NV_STATUS dmamap_src_sysmem_pages(uvm_va_block_t *va_block,
+ 
+         continue;
+ 
+-    clr_mask:
++        clr_mask:
```
Comment (ptr1337) [inline nvidia/nvidia-open-dkms/0007-Replace-PageSwapCache-for-6.12-kernel.patch:49]: What happened here?

Comment (1Naim) [reply nvidia/nvidia-open-dkms/0007-Replace-PageSwapCache-for-6.12-kernel.patch:49]: Text editor things....

Comment (1Naim) [reply nvidia/nvidia-open-dkms/0007-Replace-PageSwapCache-for-6.12-kernel.patch]: Fixed, thanks!

**Pull request merged**